smarts/core/sumo_traffic_simulation.py

```diff
@@ -206,8 +206,10 @@ def _initialize_traci_conn(self, num_retries=5):
 
         try:
             # It is mandatory to set order when using multiple clients.
+            # pytype: disable=attribute-error
             self._traci_conn.setOrder(0)
             self._traci_conn.getVersion()
+            # pytype: enable=attribute-error
```

Comment:
I think the type of `self._traci_conn` is `Optional[traci.connection.Connection]`.